test/lib/util/fast_json_encoder_test.exs

```diff
@@ -0,0 +1,61 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+defmodule Antikythera.FastJasonEncoderTest do
+  use Croma.TestCase
+  use ExUnitProperties
+
+  def validate(value) do
+    validate(value, value)
+  end
+  def validate(source, expected) do
+    {:ok, json} = FastJasonEncoder.encode(source)
+    assert json == Poison.encode!(source)
```

Comment:
In other places, We have usually used the right side of an equation as the expected value.
```
assert Poison.encode!(source) == json
```

Reply:
fixed. b0c7c48381997e5ac476ebf9b332e49eab348fd4